CHANGELOG.md

```diff
@@ -3,8 +3,9 @@
 ## master / unreleased
 
 * [CHANGE] Alertmanager now removes local files after Alertmanager is no longer running for removed or resharded user. #3910
-* [CHANGE] Alertmanager now stores local files in per-tenant folders. Files stored by Alertmanager previously are migrated to new hierarchy. Support for this migration will be removed in Cortex 1.10. #3910
```

Comment:
I've changed this from 1.10 to 1.11 because we have to keep deprecated features for 2 releases so since this will end up in 1.9 we can remove it in 1.11. Am I missing anything?

Reply:
SGTM